### DISCUSSION TITLE
Character accent options on long-press

### DISCUSSION THREAD

**Discussion by martinstork** [elementary/os#739] [Ideas]
## Prerequisites
- [x] I have searched open and closed issues for duplicates.

## Feature
**Is your feature request related to a problem? Please describe.**
By using macOS and mobile phones for a while I'm missing the character accent menu on long-press. In my opinion this has become very intuitive for most people. 

![macOS](https://support.apple.com/library/content/dam/edam/applecare/images/en_US/macos/highsierra/macos-high-sierra-text-edit-alternate-characters.jpg) _macOS example_

**Describe the solution you'd like**
When a user long presses a character (which could have accents / special characters) a pop-up would appear with the available options and corresponding numbers. Choosing an option would be done with the same numbers on the keyboard. Additionally it could be selected with a mouse click and/or navigation keys. The ESC key or a click outside the options boundary would cancel out the pop-up.

**Existing work**
I'm aware that character accents are possible with the use of the compose key. I haven't found a similar existing feature request.

**Describe alternatives you've considered**
* Have it as an opt-in or opt-out system preference under keyboard settings.
* Having the long-press activated when its still pressed or use it as a toggle.

**Additional implementation examples**
[Android](https://www.howtogeek.com/wp-content/uploads/2015/02/img_54eff94845887.png)
[iOS](https://www.howtogeek.com/wp-content/uploads/2015/02/img_54eff96f3e595.png)

**Comment by cassidyjames** [comment]
This would be _awesome_, but I believe would be completely the responsibility of GTK.

**Comment by martinstork** [comment]
> This would be _awesome_, but I believe would be completely the responsibility of GTK.

Thank you for replying. Wasn’t sure were the responsibility laid for these kind of features. Great to hear you like the idea.

**Comment by ryonakano** [comment]
Closing as out of scope as Cassidy said.
